lib/media/playhead.js

```diff
@@ -376,17 +376,18 @@ shaka.media.MediaSourcePlayhead = class {
 
     const currentTime = this.videoWrapper_.getTime();
     let seekStart = this.timeline_.getSeekRangeStart();
-    const seekEnd = this.timeline_.getSeekRangeEnd();
+    let seekEnd = this.timeline_.getSeekRangeEnd();
 
     if (seekEnd - seekStart < this.minSeekRange_) {
       seekStart = seekEnd - this.minSeekRange_;
+      seekEnd += this.minSeekRange_;
```

Comment:
Why is it necessary to make the seek range twice minSeekRange_ in size?

Reply:
Sorry for the delayed reply.
This is for test case where live manifests with no seek range.  I am not familiar with this case, and and am unsure whether I should modify the test expectations or adding a margin around the seekEnd to prevent the mechanism from taking effect in this case.

Reply:
Sorry, I don't understand what you mean.

On line 382, `seekStart = seekEnd - this.minSeekRange_`, we moved the start back so that the range size is `this.minSeekRange_`.  I don't see why you would need to double that range size by also saying `seekEnd += this.minSeekRange_`.  The existing code suffices to give the minimum range size.